tests/unit/NetworkTest.js

```diff
@@ -274,37 +274,6 @@ test('Request will not run until credentials are read from Onyx', () => {
     });
 });
 
-test('Non-retryable request will not be retried if connection is lost in flight', () => {
```

Comment:
Why are we removing this test? We should not be retrying things that are not in the write queue...?

Reply:
The way I understood it we only retry things in the sequential queue. And since everything in the sequential queue should be retryable this case shouldn't exist.

And I was thinking of adding a test that will test that the queue indeed retries all the retryable errors.

Reply:
I still don't understand why we should remove the test. It verifies the expectation that these requests will not be retried and that the response looks normal.

Reply:
>I was thinking of adding a test that will test that the queue indeed retries all the retryable errors.

That sounds good.

Reply:
> It verifies the expectation that these requests will not be retried and that the response looks normal.

The way I see it there is no such thing anymore as a non-retryable request. Retrying is a property of the sequential queue, not of a request. 
So there's no value in checking that a request will not retry itself because that no longer exists.

Maybe we could change this to 'Check that the main queue doesn't retry requests', but even that I think is unnecessary give that we didn't write any code in the main queue to retry requests.